### DISCUSSION TITLE
Por que troquei o atom pelo nano

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#347] [Perguntas e Respostas]
[hackergaucho.com/post/por_que_troquei_o_atom_pelo_nano](http://hackergaucho.com/post/por_que_troquei_o_atom_pelo_nano?via=gitfebr)

**Comment by robsonbittencourt** [comment]
Incrivelmente aqui o VSCode no Ubuntu foi liso e no Windows deu umas travadas...

Uso editores como o nano e o vim (não conheço os detalhes e as diferenças) somente quando estou em algum server e preciso editar algo pontual. 

Sinceramente não consigo me imaginar trocando um editor como Sublime, VSCode ou Atom (interface) por um que seja exclusivo no terminal. Mas sei que assim como você, muitas pessoas fizeram isso e são super produtivas.

**Comment by ghost** [comment]
@robsonbittencourt  talvez seja o meu gerenciador de janelas A principal razão de ter trocado o GUI pelo CLI é estar usando o [dwm](http://dwm.suckless.org/) que é mais focado no modo texto.
